pyoptsparse/pyOpt_history.py

```diff
@@ -193,10 +193,13 @@ def _processDB(self):
         self.DVInfo = self.read("varInfo")
         self.conInfo = self.read("conInfo")
         self.objInfo = self.read("objInfo")
+        # metadata
+        self.metadata = self.read("metadata")
+        self.optProb = self.read("optProb")
         # load names
-        self.DVNames = set(self.DVInfo.keys())
-        self.conNames = set(self.conInfo.keys())
-        self.objNames = set(self.objInfo.keys())
+        self.DVNames = set(self.getDVNames())
+        self.conNames = set(self.getConNames())
+        self.objNames = set(self.getObjNames())
 
```

Comment:
Is this also related to py3 update? why were these `.keys()` used when the other functions were available? Anyway, good you updated this

Reply:
The old implementation used the `.keys()` call, but I realized that I wrote these functions. Previously, these functions did the exact same thing, but I updated `getConNames` to skip linear constraints, and here this should be consistent also, so I switched to using these functions.